client/battle/BattleInterface.cpp

```diff
@@ -766,3 +741,11 @@ void BattleInterface::executeOnAnimationCondition( EAnimationEvents event, bool
 {
 	awaitingEvents.push_back({action, event, state});
 }
+
+void BattleInterface::setBattleQueueVisibility(bool visible)
+{
+	if(visible)
+		windowObject->showQueue();
+	else
+		windowObject->hideQueue();
+}
```

Comment:
Obviously minor, but please keep an empty line in the end of file
```suggestion
}

```

Reply:
done